### PR TITLE
Add migrated database stdlibs to distribution

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
$subject after the effort in https://github.com/ballerina-platform/ballerina-lang/issues/25085

## Approach
1. Add the database modules as std lib dependencies

## Automation tests
 - Unit tests - N/A
 - Integration tests - Already added

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

## Migrations (if applicable)
`ballerina/mysql` -> `ballerinax/mysql`

## Test environment
JDK 8.0.231, H2 1.4.200, MySQL 8.0

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -253,6 +278,9 @@ subprojects {
         ballerinaStdLibs "org.ballerinalang:nats-ballerina:${stdlibNatsVersion}"
         ballerinaStdLibs "org.ballerinalang:system-ballerina:${stdlibSystemVersion}"
         ballerinaStdLibs "org.ballerinalang:runtime-ballerina:${stdlibRuntimeVersion}"
+        ballerinaStdLibs "org.ballerinalang:sql-ballerina:${stdlibSqlVersion}"
+        ballerinaStdLibs "org.ballerinalang:java.jdbc-ballerina:${stdlibJdbcVersion}"
+        ballerinaStdLibs "org.ballerinalang:mysql-ballerina:${stdlibMySqlVersion}"
```
Comment (BuddhiWathsala) [inline build.gradle:283]: Shouldn't we need to specify version global variables like this?
```
stdlibSystemVersion = project.stdlibSystemVersion
```

File: examples/jdbc-query-operation/jdbc_query_operation.bal [hunk 2]
```diff
@@ -25,12 +25,6 @@ function simpleQuery(jdbc:Client jdbcClient) {
         io:println(e);
     }
 
-    // In general cases, the stream will be closed automatically
-    // when the stream is fully consumed or any error is encountered.
-    // However, in case if the stream is not fully consumed, the stream
-    // should be closed specifically.
-    e = resultStream.close();
```
Comment (BuddhiWathsala) [inline examples/jdbc-query-operation/jdbc_query_operation.bal:32]: Any reason to remove this?

**Review by BuddhiWathsala — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (niveathika) [reply build.gradle:283]: Is that necessary? The local build wee passing so I didn't add those.

File: examples/jdbc-query-operation/jdbc_query_operation.bal (hunk 2 above)
Comment (niveathika) [reply examples/jdbc-query-operation/jdbc_query_operation.bal:32]: Related to discussions in https://github.com/ballerina-platform/ballerina-distribution/issues/730

**Review by BuddhiWathsala — APPROVED** [review]

**Pull request merged**